templates/update_coregistration_params.ipynb

```diff
@@ -136,14 +137,12 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "ae5ccc65-f555-47e6-983e-d8944ac42bf1",
+   "id": "f126bc6a-4695-4722-9d73-b9dc66bb7195",
    "metadata": {},
    "outputs": [],
    "source": [
-    "tiling_utils.write_coreg_params(\n",
-    "    coreg_params,\n",
-    "    fiducial_info['name']\n",
-    ")"
+    "with open(os.path.join('..', 'toffy', 'stage_optical_coreg_params.json'), 'w') as cp:\n",
+    "    json.dump(coreg_params, cp)"
```

Comment:
Currently, this file name, `'stage_optical_coreg_params.json'`, is the default setting. Should we allow the user to specify different names or keep this constant?